web-locks/resources/request-around-navigation.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head><meta charset="utf-8"></head>
+<body>
+<script>
+'use strict';
+(async () => {
+  const [uuid, phase] = location.search.split('_');
```

Comment:
This works, but is a bit less clear than explicit query params. You could parse those using

```js
const url = new URL(self.location.href);
const lockName = url.searchParams.get('lock');
const state = url.searchParams.get('state');
```